src/utils/index.js

```diff
@@ -198,6 +199,44 @@ const formatCollaboratorOptions = (options) => (
   options.map(({ name = '', uuid, image: { original: { url } = { url: '' } } }) => ({ name, uuid, image_url: url }))
 );
 
+function courseTagObjectsToSelectOptions(allCourseTags) {
+  /*  transform an array of course tag objects e.g
+    [
+      {
+        name: 'mba',
+        value: 'mba'
+      },
+      {
+        name: 'mba-gmat',
+        value: 'mba-gmat'
+      },
+    ]
+    to a format expected by ReduxFormCreatableSelect i.e
+    [
+      {
+        label: 'mba',
+        value: 'mba'
+      },
+      {
+        label: 'mba-gmat',
+        value: 'mba-gmat'
+      }
+    ]
+  */
+  return allCourseTags.map(tag => ({
+    label: tag.value,
+    value: tag.value,
+  })).filter(x => x.value);
+}
+
+const loadOptions = (inputValue, callback) => DiscoveryDataApiService.fetchCourseTags(inputValue)
+  .then((response) => {
+    callback(courseTagObjectsToSelectOptions(response.data));
+  })
+  .catch(() => {
+    callback(null);
+  });
```

Comment:
There seems to be some rendering issue on Github for this piece of code. Can you double-check if the multi-line comment string is affecting this?

Reply:
I have tried removing the multiline comment, but the rendering remains unchanged.
<img width="606" alt="image" src="https://user-images.githubusercontent.com/78806673/231449116-154a6e0c-b810-4b77-9039-8069138daa38.png">